button.css

```diff
@@ -0,0 +1,98 @@
+.button {
+    background-color: rgb(183, 231, 183);
+    border: none;
+    padding: 16px 16px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 10px 2px;
+    transition-duration: 0.4s;
+    cursor: pointer;
+}
+
+.button1 {
+    background-color: rgb(39, 243, 12);
+    color: black;
+    border: 2px solid;
+}
+
+.button2 {
+    background-color: rgb(39, 235, 235);
+    color: black;
+    border: 2px solid #0f0f0e;
+    border-radius: 8%;
+}
+
+.button2:hover {
+    background-color: #f552ea;
+    color: rgb(55, 13, 241);
+}
+
+.button3 {
```

Comment:
Instead of button1, button2 . Naming can be given as button-primary or button-blue.